app/components/application_record_card/view.rb

```diff
@@ -29,8 +31,22 @@ def route
       record.record_type.humanize
     end
 
+    def updated_at
+      date_stamp = record.updated_at.presence || record.created_at
```

Comment:
`record.updated_at` should always be present as it gets populated on create and update. This means you can get of the whole line and just use `record.updated_at`. You might need to update the specs accordingly.